lms/djangoapps/teams/static/teams/js/collections/base.js

```diff
@@ -27,7 +27,7 @@
                     var self = this,
                         deferred = $.Deferred();
                     if (force || this.isStale) {
-                        this.fetch()
+                        this.setPage(1)
```

Comment:
I'm not exacly sure how `Backbone.paginator` interprets `fetch`, but I'd imagine that it would refresh the current page of results, no? If calling `setPage(1)` resets to the first page of the collection, then aren't we changing behavior here?